packages/functional-tests/src/functional-tests.local.spec.ts

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { CredentialType, registerAzureServicesToContainer } from 'azure-services';
+import { GuidGenerator, ServiceConfiguration, setupRuntimeConfigContainer, System } from 'common';
+import * as dotenv from 'dotenv';
+import { Container } from 'inversify';
+import { isEmpty } from 'lodash';
+import { ConsoleLoggerClient, GlobalLogger, Logger } from 'logger';
+import { OnDemandPageScanRunResultProvider, RunState, ScanResultResponse, ScanRunResultResponse } from 'service-library';
+import { A11yServiceClient, A11yServiceCredential } from 'web-api-client';
+
+import { TestEnvironment } from './common-types';
+import { functionalTestGroupTypes, TestGroupName } from './functional-test-group-types';
+import { TestRunner } from './runner/test-runner';
+import { TestContextData } from './test-group-data';
+import { FunctionalTestGroup } from './test-groups/functional-test-group';
+
+// tslint:disable: mocha-no-side-effect-code no-any no-unsafe-any mocha-unneeded-done strict-boolean-expressions
+
+describe('functional tests', () => {
```

Comment:
In general would be better to have this implemented as a generic e2e tests group runner similar to what we have in health orchestration worker

Reply:
Reuse the health REST API result can be part of this, so we can unify implementation and do not fork e2e runners

Reply:
Agreed. that could be in a future PR?